templates/resources/deploy-task-engine/bin/run-in-container.sh.j2

```diff
@@ -139,6 +139,36 @@ function store_license {
   fi
 }
 
+function escape_special_characters() {
+    printf '%s\n' "$1" | sed -e 's#!#\\!#g' \
+                              -e 's#%#\\%#g' \
+                              -e 's#\^#\\^#g' \
+                              -e 's#&#\\&#g' \
+                              -e 's#\*#\\*#g' \
+                              -e 's#(#\\(#g' \
+                              -e 's#)#\\)#g' \
+                              -e 's#-#\\-#g' \
+                              -e 's#_#\\_#g' \
+                              -e 's#=#\\=#g' \
+                              -e 's#<#\\<#g' \
+                              -e 's#>#\\>#g' \
+                              -e 's#?#\\?#g' \
+                              -e 's/#/\\#/g' \
+                              -e 's#\$#\\$#g' \
+                              -e 's#+#\\+#g' \
+                              -e 's#\[#\\[#g' \
+                              -e 's#\]#\\]#g' \
+                              -e 's#{#\\{#g' \
+                              -e 's#}#\\}#g'
+}
```

Comment:
Copilot says that following is more optimized:

`function escape_special_characters() {
    printf '%s\n' "$1" | sed -e 's/[#%^&*()\-_=<>?\/$+[\]{}]/\\&/g'
}`

What do you think?

Reply:
Or we can say in doc that some characters needs to be escaped

Reply:
and what if it is already entered escaped?